ydb/docs/en/core/integrations/flyway.md

```diff
@@ -0,0 +1,234 @@
+# Migrating {{ ydb-short-name }} data schemas with the Flyway migration tool
+
+## Introduction {#introduction}
+
+[Flyway](https://documentation.red-gate.com/fd/) is an open-source database migration tool. It strongly favors simplicity and convention over configuration. It has extensions for various database management systems (DBMS), including {{ ydb-short-name }}.
```

Comment:
If you follow this link, the most prominent product is Flyway Desktop. Shouldn't it be covered as well?

Reply:
The Amazon CloudFront distribution is configured to block access from your country. We can't connect to the server for this app or website at this time. There might be too much traffic or a configuration error. Try again later, or contact the app or website owner.
If you provide content to customers through CloudFront, you can find steps to troubleshoot and help prevent this error by reviewing the CloudFront documentation.

Reply:
I can't download the desktop application, it seems to me it's an unused part.